Mathlib/LinearAlgebra/InvariantBasisNumber.lean

```diff
@@ -56,6 +72,10 @@ variants) should be formalized.
 
 * https://en.wikipedia.org/wiki/Invariant_basis_number
 * https://mathoverflow.net/a/2574/
+* Lam, T. Y. *Lectures on Modules and Rings*
```

Comment:
Add these references to the bibtex file.

Reply:
Added.